CHANGELOG.md

```diff
@@ -2,9 +2,51 @@
+  The feature gate `extension.filestorage.replaceUnsafeCharacters` is now removed.
+- (Contrib) `vcenterreceiver`: vcenterreceiver replaces deprecated packet metrics by removing them and enabling by default the newer ones. (([#32929](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/32929)),([#32835](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/32835))
+  Removes the following metrics: `vcenter.host.network.packet.errors`, `vcenter.host.network.packet.count`, and
+  `vcenter.vm.network.packet.count`. 
+
+  Also enables by default the following metrics: `vcenter.host.network.packet.error.rate`,
+  `vcenter.host.network.packet.rate`, and `vcenter.vm.network.packet.rate`.
+- (Core) `envprovider`: Restricts Environment Variable names.  Environment variable names must now be ASCII only and start with a letter or an underscore, and can only contain underscores, letters, or numbers. ([#9531](https://github.com/open-telemetry/opentelemetry-collector/issues/9531))
+- (Core) `confighttp`: Apply MaxRequestBodySize to the result of a decompressed body [#10289](https://github.com/open-telemetry/opentelemetry-collector/pull/10289)
+
+  When using compressed payloads, the Collector would verify only the size of the compressed payload. 
+  This change applies the same restriction to the decompressed content. As a security measure, a limit of 20 MiB was added, which makes this a breaking change. 
+  For most clients, this shouldn't be a problem, but if you often have payloads that decompress to more than 20 MiB, you might want to either configure your
+  client to send smaller batches (recommended), or increase the limit using the MaxRequestBodySize option.
+
+
+### 🧰 Bug fixes 🧰
+
+- (Splunk) `discovery`: Fix crashing collector if discovered mongodb isn't reachable in Kubernetes ([#4911](https://github.com/signalfx/splunk-otel-collector/pull/4911))
+- (Contrib) `podmanreceiver`: add scraper's shutdown method ([#29994](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/29994))
+- (Contrib) `awsxrayexporter`: Fix the DB subsegment(client span) name with JDBC conn string starts with "jdbc:" ([#33225](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33225))
+- (Contrib) `exp/metrics`: fixes staleness.Evict such that it only ever evicts actually stale metrics ([#33265](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33265))
+- (Contrib) `receiver/mysql`: Remove the order by clause for the column that does not exist ([#33271](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33271))
+- (Contrib) `influxdb(exporter|receiver)`: remove Metric flags field to/from InfluxDB conversion ([#29896](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/29896))
```

Comment:
```suggestion
```
We don't have influxdb components in our distro.

Reply:
removed